src/docs/asciidoc/developer-manual/develop-cloud-services/initialization-code.adoc

```diff
@@ -0,0 +1,27 @@
+:relative-path: ../../
+include::{docdir}/variables.adoc[]
+
+ZetaPush provides you the possibility to add some initialization code in your _custom cloud service_.
```

Comment:
We should explain why this is needed. We should explain that as ZetaPush manages scalability and redundancy, there may have several workers that start at the same time. And if you initialize some data or configure some cloud services at the same time, it may have conflicts or duplicated data. To avoid that ZetaPush provides a way to initialize code that will ensure that is executed by only one worker. We should then explain that initializing data or configuring services must not be done in constructor. Then we show how to do that.